content/modules/modules-quickstart.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Modules Quick Start
+description:
+weight: 1
+alwaysopen: false
+categories: ["Modules"]
+---
+To quickly get access to a Redis Enterprise database with a module,
+you can sign up for a free Redis Cloud database and enable the module that you want to try out.
+
+The modules that are currently available are:
+
+- [RediSearch]({{< relref "/modules/redisearch/_index.md" >}})
+- [RedisGraph]({{< relref "/modules/redisgraph/_index.md" >}})
+- [RedisBloom]({{< relref "/modules/redisbloom/_index.md" >}})
+- [RedisJSON]({{< relref "/modules/redisjson/_index.md" >}})
+- [RedisTimeSeries]({{< relref "/modules/redistimeseries/_index.md" >}})
+
+{{< note >}}
+Modules are currently available as a public preview in AWS region `ap-south-1`.
+They will be available in other providers and regions after the public preview period is finished.
+{{< /note >}}
+
+The steps for getting up and running with a database that has a module enabled are:
+
+1. Create a new subscription.
+1. Create a new database with the module enabled.
+1. Connect to the database.
+
+## Step 1: Create a New Subscription
+
+To create a new subscription:
+
+1. In the Redis Cloud menu, click **Subscriptions**.
+1. At the bottom of the page, click ![Add](/images/rs/icon_add.png#no-click "Add").
+1. Select your subscription configuration:
+    1. For the cloud provider, select **Amazon AWS**.
+    <!-- , **Microsoft Azure**, **Google Cloud Platform** -->
+    1. For the region that you want the subscription to use, select **ap-south-1**.
+    1. In the Redis Cloud service levels, select the Redis Cloud Essentials **30MB/1 Database** level.
+    1. Enter a name for the subscription.
+1. Click **Create**.
+
+The subscription shows a "Pending" status and takes approximately 10 to 15 minutes to provision.
```

Comment:
@Aviad-Abutbul @bmansheim Is this the case for modules on essentials databbases?

Reply:
Maybe a minute. I changed it.